cache/src/main/java/org/jclouds/karaf/cache/utils/CacheUtils.java

```diff
@@ -61,6 +63,28 @@ public void removedService(ServiceReference reference, Object service) {
         };
     }
 
+    public static <T> ServiceTracker createServiceCacheTracker(final BundleContext context, final TypeToken<T> serviceToken, final CacheManager<T> cacheManager) {
+        return new ServiceTracker(context, serviceToken.getRawType().getName(), null) {
+
+            @Override
+            public Object addingService(ServiceReference reference) {
+                Object service = super.addingService(reference);
+                if (serviceToken.isAssignableFrom(service.getClass())) {
+                    cacheManager.bindService((T) service);
+                }
+                return service;
+            }
+
+            @Override
+            public void removedService(ServiceReference reference, Object service) {
+                if (serviceToken.isAssignableFrom(service.getClass())) {
+                    cacheManager.unbindService((T) service);
+                }
+                super.removedService(reference, service);
+            }
+        };
+    }
```

Comment:
Yes, all the files were formatted that way and I didn't want to reformat everything in this PR.